src/include/platform/OTARequestorDriver.h

```diff
@@ -54,7 +56,23 @@ enum class UpdateNotFoundReason
 {
     Busy,
     NotAvailable,
-    UpToDate
+    UpToDate,
+    ConnectionFailed,
+};
+
+enum class OTARequestorIncomingEvent
+{
+    AnnouncedOTAProviderReceived,
```

Comment:
nit: the individual values typically start with k

Reply:
Fixed